cau12.md

```diff
@@ -0,0 +1,8 @@
+###12. Lấy blog được tạo bởi user mà user này không có bất kỳ comment ở blog
+select bl.*
+from `train-mysql`.user as us inner join `train-mysql`.blog as bl on us.id=bl.user_id
```

Comment:
Chỗ này không join đâu, mình đứng từ blog rồi where user_id not in (select distinct user_id from comment where  target_table="blog")